crates/matrix-sdk-ui/src/notification_client.rs

```diff
@@ -632,6 +632,20 @@ impl NotificationEvent {
             NotificationEvent::Invite(ev) => &ev.sender,
         }
     }
+
+    fn thread_id(&self) -> Option<OwnedEventId> {
```

Comment:
Let's just add a small docstring explaining this function, i.e when it's useful.